CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
 
+### ⚠️ Breaking Changes
+- `Atomic.get()` no longer returns an optional type if the wrapped type itself is not optional. `Atomic.init(_:)` now always requires the initial value [#241](https://github.com/GetStream/stream-chat-swift/issues/241).
+
```

Comment:
Maybe we should merge this into `release/3.0` since it's breaking and touches a lot of different parts?

Reply:
I'd like to merge it to `master`. Let me redo it in a bit more compatible way.